modules/stepper/demo/stepper.html

```diff
@@ -0,0 +1,127 @@
+                </tr>
+                <tr>
+                    <td>lx-complete*</td>
+                    <td><code>function</code></td>
+                    <td></td>
+                    <td>Complete function called when stepper is complete.</td>
+                </tr>
+                <tr>
+                    <td>lx-is-linear</td>
+                    <td><code>boolean</code></td>
+                    <td><code>true</code></td>
+                    <td>Define the stepper type.</td>
+                </tr>
+                <tr>
+                    <td>lx-labels</td>
+                    <td><code>object</code></td>
+                    <td><code>{ 'back': 'Back', 'cancel': 'Cancel', 'continue': 'Continue', 'optional': 'Optional' }</code></td>
+                    <td>The buttons and the optionnal step labels.</td>
+                </tr>
+                <tr>
+                    <td>lx-layout</td>
+                    <td><code>string</code></td>
+                    <td><code>horizontal</code></td>
+                    <td>Define the stepper layout. Options: <code>horizontal</code>, <code>veritcal</code></td>
+                </tr>
+            </tbody>
+        </table>
+    </lx-component-attributes>
+
+    <lx-component-attributes lx-title="Step attributes">
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter</th>
+                    <th>Type</th>
+                    <th>Default</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+
+            <tbody>
+                <tr>
+                    <td>lx-feedback</td>
+                    <td><code>string</code></td>
+                    <td></td>
+                    <td>The feedback string displayed during the next step loading (Only if horizontal layout).</td>
+                </tr>
+                <tr>
+                    <td>lx-is-editable</td>
+                    <td><code>boolean</code></td>
+                    <td><code>false</code></td>
+                    <td>Define if the step is editable. If so, submitting the step after the first time trigger the complete function.</td>
+                </tr>
+                <tr>
+                    <td>lx-is-optional</td>
+                    <td><code>boolean</code></td>
+                    <td><code>false</code></td>
+                    <td>Define if the step is optional. If so, it's possible to jump to the next step without submitting (Only if linear stepper).</td>
+                </tr>
+                <tr>
+                    <td>lx-label*</td>
+                    <td><code>string</code></td>
+                    <td></td>
+                    <td>The step label displayed in the navigation.</td>
+                </tr>
+                <tr>
+                    <td>lx-submit</td>
+                    <td><code>function</code></td>
+                    <td></td>
+                    <td>Submit function called on step submit if valid.</td>
```

Comment:
`... Returns a Promise object that is resolved with the index of the next step to navigate to or a Promise object that is rejected with the error message.`